packages/box/lib/utils/unbox.ts

```diff
@@ -148,7 +148,12 @@ function installBoxDependencies({ hooks }: boxConfig, destination: string) {
   const postUnpack = hooks["post-unpack"];
 
   if (postUnpack.length === 0) return;
-  execSync(postUnpack, { cwd: destination, stdio: "ignore" });
+
+  spawnSync(postUnpack, {
+    cwd: destination,
+    shell: true,
+    stdio: ["ignore", process.stdout, process.stderr]
```

Comment:
Ignore `stdin` and pipe `stdout` / `stderr` to parent process. I also piped `stdout` because - after looking at some boxes - most of them use `--loglevel=error` to elevate the logging to `stderr`.

`execSync` was previously left at its default config that didn't pipe `stdout`, now that it is with `spawnSync` the log level flag is not required anymore.